In-progress

Attempt to fix mouseover flicker for imagebutton and imagemap with
alpha detection

Index: renpy/renpy/gl/gldraw.pyx
===================================================================
--- renpy.orig/renpy/gl/gldraw.pyx
+++ renpy/renpy/gl/gldraw.pyx
@@ -528,6 +528,14 @@ cdef class GLDraw:
             self.always_opaque = True
             gltexture.use_gles()
 
+        elif renpy.emscripten:
+            # give back control to browser regularly
+            self.redraw_period = 0.1
+            # True messes alpha imagemap/imagebutton mouseover
+            self.always_opaque = False
+            # WebGL is GLES
+            gltexture.use_gles()
+
         else:
             gltexture.use_gl()
 
@@ -637,7 +645,7 @@ cdef class GLDraw:
         # ANGLE from working with fbo on Windows.
 
         use_fbo = (
-            renpy.ios or renpy.android or (EGL and not ANGLE) or
+            renpy.ios or renpy.android or renpy.emscripten or (EGL and not ANGLE) or
             use_subsystem(
                 glrtt_fbo,
                 "RENPY_GL_RTT",
